ui/src/components/BaseFormView/BaseFormView.tsx

```diff
@@ -142,8 +142,10 @@ class BaseFormView extends PureComponent<BaseFormProps, BaseFormState> {
         this.groupEntities = [];
         this.endpoint =
             props.mode === MODE_EDIT || props.mode === MODE_CONFIG
-                ? `${this.props.serviceName}/${encodeURIComponent(this.props.stanzaName)}`
-                : `${this.props.serviceName}`;
+                ? `${encodeURIComponent(this.props.serviceName)}/${encodeURIComponent(
+                      this.props.stanzaName
+                  )}`
+                : `${encodeURIComponent(this.props.serviceName)}`;
```

Comment:
we probably should also include more restrict validation for service names in global config, something similar as we got for meta.name
as `"name": "custom_abc/asd",`
is still valid name under json schema,

it breaks during build as python can not map / for file route, but would be good to mark it earlier